tests/runtest.proj

```diff
@@ -412,6 +412,21 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
 
   </Target>
 
+  <Target Name="RunPerfTests" Condition="'$(Performance)'=='true'">  
+    <Message Text="Executing steps for perf tests" Importance="High"/>  
+    
+    <!-- generate project.json for runtime dependency -->  
+    <MSBuild Projects="$(MSBuildProjectFile)"  
+             Targets="CreateTestRuntimeJsonFile"/>  
+     
+    <!-- Package the product -->  
+    <MSBuild Projects="$(MSBuildProjectFile)" Targets="CreateTestOverlay" />  
```

Comment:
If I understand correctly the runtime dependencies get copied over from here
https://github.com/dotnet/coreclr/blob/43cc989069c400c939f1d4df878876523bf336ed/tests/publishdependency.targets#L15
If I don't call this target the assembly list generated has empty "corerun" element